anxcloud/struct_vsphere_locations_test.go

```diff
@@ -0,0 +1,72 @@
+package anxcloud
```

Comment:
could you move that to `struct_locations_test.go` file to have each location in one file? That would help later with refactoring.

Reply:
done